### PR TITLE
Logging typos fixed, maybe one error code definition too

### LINKED ISSUE #12051: Fix typos in logging messages, possible leftover bug from #11765, and remove two unused functions in WMCore/Database
**Impact of the new feature**
While looking for typos in most of the python code (that's specifically printed out), I found a seemingly unfixed part of the PR #11765 , see https://github.com/dmwm/WMCore/pull/11765#discussion_r1369296362 which still seems to be there: https://github.com/dmwm/WMCore/compare/master...oljemark:WMCore:custom-typo
Should this be fixed, or was this a DB field that was created with a certain typo and now has to be kept as is?

I also found that WMCore/Database/CmsCouch.py had two seemingly totally unused functions, getPrevisousRevision() & non_rotating_commit(), that were suspected as such already in #6928 .

The log text typos I found are here: https://github.com/dmwm/WMCore/compare/master...oljemark:WMCore:typo and error message & handler watchword typos here https://github.com/dmwm/WMCore/compare/master...oljemark:WMCore:twiki-codes
In the latter case, can I assume WMCore/Agent/Flow/DefaultFlow.py is not used anywhere, but just an example to be filled in by the user?

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
If you think any of these corrections could be acceptable, I could open up to three PR's based on the branches above, and https://github.com/dmwm/WMCore/compare/master...oljemark:WMCore:remove2old , one each.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by oljemark**
Fixes #12051 

#### Status
not-tested 

#### Description
Fixes some logging output typos, that seem not have been hardcoded into parsing code elsewhere on github (except for the second commit with 'woud's ).
Is WMCore/Agent/Flow/DefaultFlow.py used at all? I only saw it in a help output from the flow generation as a default file to modify to your own needs, not run as is? If you want, I could cherry-pick the commits from https://github.com/dmwm/WMCore/compare/master...oljemark:WMCore:twiki-codes to this PR - the error code typo fix would also need simultaneous fix in the TWiki.

#### Is it backward compatible (if not, which system it affects?)
MAYBE

### CODE REVIEW

**Comment by amaltaro** [comment]
test this please

File: src/python/WMComponent/JobCreator/JobCreatorPoller.py [hunk 1]
```diff
@@ -466,7 +466,7 @@ def pollSubscriptions(self):
             except Exception as ex:
                 msg = "Had failure loading generators for subscription %i\n" % (subscriptionID)
                 msg += "Exception: %s\n" % str(ex)
-                msg += "Passing over this error.  It will reoccur next interation!\n"
+                msg += "Passing over this error.  It will reoccur next interaction!\n"
```
Comment (amaltaro) [inline src/python/WMComponent/JobCreator/JobCreatorPoller.py:469]: This is actually supposed to be "iteration".

**Review by amaltaro — CHANGES_REQUESTED** [review]
These changes are looking good to me as well, but there is one module that needs to be updated (please see my comment along the code).

Once you make that change, could you please squash the new commit into the first one? In case you need some directions on that process, I find this https://steveklabnik.com/writing/how-to-squash-commits-in-a-github-pull-request a good source. Thank you again!

**Comment by oljemark** [comment]
Hello. Fixed now, if you meant for the 2nd commit to remain separate.

**Comment by amaltaro** [comment]
Great, thanks Fredrik!
Ideally, the 2nd commit would have only test/* changes, as mentioned in the contributing guidelines. But I appreciate your prompt fix and it is good to go!

**Review by amaltaro — APPROVED** [review]

**Pull request merged**